_data/simple-icons.json

```
@@ -2156,6 +2156,11 @@
             "hex": "2B579A",
             "source": "https://www.office.com"
         },
+        {
+            "title": "Microsoft Teams",
+            "hex": "2B579A",
```

Excellent PR, @bonez0607 . 
Once again, thank you for taking the time to contribute to Simple Icons. 

I also cast Option 2 out of the suggested icon styles from the linked issue.  

Other than that, the color should be `#6264A7` as it consistently appears throughout the linked website and the main app (at https://teams.microsoft.com). 
It is also listed in as one of the color in the official palette from the [Microsoft Teams brand guide](https://docs.microsoft.com/en-us/microsoftteams/platform/resources/design/components/color) (for their app UI anyway but I think it's safe to say that it is the main color).  

```suggestion
            "hex": "6264A7",
```